.github/CODEOWNERS

```diff
@@ -52,7 +50,6 @@ azure-pipelines.yml @coq/ci-maintainers
 # Secondary maintainer @mattam82
 
 /dev/doc/         @Zimmi48
-# Secondary maintainer @maximedenes
```

Comment:
Move this file to @coq/doc-maintainers ownership if it's fine with @cpitclaudel. Otherwise, I think @jfehrle would probably be OK to be secondary maintainer.